README.md

```diff
@@ -46,3 +46,7 @@ The protocol buffers library is distributed under the same BSD license as
 Google's protocol buffers. See its
 [README](https://github.com/protocolbuffers/protobuf/blob/master/README.md) and
 [LICENSE](https://github.com/protocolbuffers/protobuf/blob/master/LICENSE).
+
+## Running on GNU/Linux ##
+
+For GNU/Linux, visit [Darling](https://github.com/darlinghq/darling)
```

Comment:
Please add a bit more information, since many j2objc developers don't already work with Linux, something like:

To build and run on GNU/Linux, install the [Darling project](https://github.com/darlinghq/darling), then following its [Compile and Run a Program](https://wiki.darlinghq.org/what_to_try#compile_and_run_a_program) example. Please note that j2objc is only supported on IOS/macOS.